fullstory-segment-middleware/src/test/java/com/fullstorydev/fullstory_segment_middleware/FSSuffixedPropertiesTest.java

```diff
@@ -262,6 +262,14 @@ public void pluralizeAllArrayKeysInMap_HasArrayInMap() {
         Assert.assertEquals(output, fsSuffixedProperties.suffixedProperties);
     }
 
+    @Test
+    public void getSuffixedProperties_ReturnsProperties() {
+        Map<String, Object> input = new HashMap<>();
+        input.put("key_strs", new ArrayList<>(Arrays.asList("val1","val2")));
+        fsSuffixedProperties.suffixedProperties = input;
+        Assert.assertEquals(input, fsSuffixedProperties.getSuffixedProperties());
```

Comment:
I'm probably missing something, but are you testing whether a property accessor on on the `fsSuffixedProperties` instance returns the value that was set?

Reply:
Oops good catch! this was a test(for the getter) when I played around with the coverage report. Should probably delete this test.